cluster/src/main/java/com/hartwig/pipeline/tertiary/lilac/Lilac.java

```diff
@@ -61,19 +65,85 @@ public String namespace() {
     }
 
     @Override
-    public List<BashCommand> commands(final SomaticRunMetadata metadata) {
+    public List<BashCommand> tumorReferenceCommands(final SomaticRunMetadata metadata)
+    {
         String slicedRefBam = VmDirectories.outputFile(metadata.reference().sampleName() + ".hla.bam");
         String slicedTumorBam = VmDirectories.outputFile(metadata.tumor().sampleName() + ".hla.bam");
-        return List.of(new LilacBamSliceCommand(resourceFiles, getReferenceBamDownload().getLocalTargetPath(), slicedRefBam),
-                new LilacBamIndexCommand(slicedRefBam),
-                new LilacBamSliceCommand(resourceFiles, getTumorBamDownload().getLocalTargetPath(), slicedTumorBam),
-                new LilacBamIndexCommand(slicedTumorBam),
-                new LilacCommand(resourceFiles,
-                        metadata.tumor().sampleName(),
-                        slicedRefBam,
-                        slicedTumorBam,
-                        purpleGeneCopyNumber.getLocalTargetPath(),
-                        purpleSomaticVariants.getLocalTargetPath()));
+
+        List<BashCommand> commands = Lists.newArrayList();
```

Comment:
This also seems like a subjective refactor. It's not really better or worse than the old way, just different. In general we break down the stages in terms of responsibilities:
- The stage wires things together (input, output, commands)
- Individual command classes define the commands

These new stages are now inconsistent with all the rest and will start to create two camps within the codebase.